demos/dermatology/.vscode/launch.json

```diff
@@ -74,6 +74,20 @@
                 "webpack:///*": "/*"
             }
         },
+        {
+            "name": "Leaflet.Heat",
+            "type": "chrome",
+            "request": "launch",
+            "url": "file:///${workspaceRoot}/index.html?src/map/Leaflet.Heat",
+            "runtimeArgs": [
+                "--allow-file-access-from-files"
+            ],
+            "webRoot": "${workspaceRoot}",
+            "sourceMapPathOverrides": {
+                "webpack:///./*": "${workspaceRoot}/*",
+                "webpack:///*": "/*"
```

Comment:
What do these `sourceMapPathOverrides` entries provide?

Reply:
Cut and paste - but if we webpacked the sources, it should resolve the TS files to webpacked bundle